app/src/main/res/layout/settings_activity.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Does this need to be two nested layouts? I think just listing the `FragmentContainerView` directly would be more efficient.

Reply:
As far as I know, it's supposed to be two layouts. FragmentContainerView actually extends FrameLayout, so it's meant to be used in the same fashion.

I will test it anyway when I get home, cause I'm interested to know what would happen (I'd expect a crash, but the log would be interesting).

Reply:
@ByteHamster Wow, I'm actually surprised that FragmentContainerView by itself works! However, if you were to add a toolbar or something like that, you would have to nest it.